.gitignore

```diff
@@ -1,7 +1,8 @@
 
 # Directories/files that may be generated by this project
 node_modules/
-dist
+/dist
```

Comment:
I adjusted these a bit so that dist still works as before but lets us check in the action's dist folder which is the default directory that `@vercel/ncc` compiles into